browser/ui/views/side_panel/playlist/playlist_side_panel_coordinator.cc

```diff
@@ -33,8 +38,26 @@ void PlaylistSidePanelCoordinator::CreateAndRegisterEntry(
 }
 
 std::unique_ptr<views::View> PlaylistSidePanelCoordinator::CreateWebView() {
-  return std::make_unique<PlaylistSidePanelWebView>(&GetBrowser(),
-                                                    base::RepeatingClosure());
+  const bool should_create_contents_wrapper = !contents_wrapper_;
+  if (should_create_contents_wrapper) {
+    contents_wrapper_ =
+        std::make_unique<BubbleContentsWrapperT<playlist::PlaylistUI>>(
+            GURL(kPlaylistURL), GetBrowser().profile(), 0,
```

Comment:
I found that passing `0`  causes crash when I'm trying to launch task manager while playlist panel is opened. It seems we should pass valid string resource id instead of `0`.
```
[125609:125609:0928/092919.330778:FATAL:resource_bundle.cc(1172)] Check failed: !data.empty(). Unable to find resource: 0. If this happens in a browser test running on Windows, it may be that dead-code elimination stripped out the code that uses the resource, causing the resource to be stripped out because the resource is not used by chrome.dll. See https://crbug.com/1181150.
#0 0x7f9160987732 base::debug::CollectStackTrace()
#1 0x7f916087bd63 base::debug::StackTrace::StackTrace()
#2 0x7f916089ba0b logging::LogMessage::~LogMessage()
#3 0x7f916089c69e logging::LogMessage::~LogMessage()
#4 0x7f915fb9fcd2 ui::ResourceBundle::GetLocalizedStringImpl()
#5 0x7f915fb9f965 ui::ResourceBundle::GetLocalizedString()
#6 0x7f915fb82228 l10n_util::GetStringUTF16()
#7 0x55b0bcd3fc41 task_manager::ToolTask::ToolTask()
#8 0x55b0bcd3fbdc task_manager::ToolTag::CreateTask()
#9 0x55b0bcd40724 task_manager::WebContentsTaskProvider::WebContentsEntry::CreateTaskForFrame()

```

Reply:
Well spotted! Fixed
<img width="361" alt="image" src="https://user-images.githubusercontent.com/5474642/192765075-52ab1c9d-ac97-4567-842e-dc4758100ac3.png">